File: frontend/src/constants/CustomerContact.ts
import { Dayjs } from "dayjs";

export type CustomerContact = {
  
  id: number;
  value: string;
  allowNotification: boolean;
  rTypeId: number;
  organizationId: number;
};


export type CustomerContactCreateModel = {
  
  id: number;
  value: string;
  allowNotification: boolean;
  rTypeId: number;
  organizationId: number;
};

File: backend/Domain/Entities/CustomerContact.cs

namespace Domain.Entities
{
    public class CustomerContact : BaseLogDomain, IBaseDomain
    {
        public int Id { get; set; }
		public string Value { get; set; }
		public bool? AllowNotification { get; set; }
		public int? RTypeId { get; set; }
		public int OrganizationId { get; set; }
		
    }
}